content/guidedLearning/guideForUnityDevs.md

```diff
@@ -0,0 +1,83 @@
+- https://github.com/BabylonJS/BabylonNative
+- https://github.com/BabylonJS/BabylonReactNative
+
+# Why Babylon is amazing
+
+So, to me, Babylon is wild. It's all open-source and maintained by some very talented people. The documentation is great (it can lag a bit behind the actual implementations though), the responses on the forum are just unreasonably quick (https://forum.babylonjs.com/), and I've been lucky enough that the developers will take time to answer any and all questions I've had about Babylon React Native. Compare that to the current state of Unity, where you can still build great stuff with it, but let's be honest, Unity is a big untouchable impersonal monolith. I don't feel like the developers working for Unity have a lot of personal investment in the work they do. And that's inevitable, but the charm of Babylon is that it's not this big monolith. It's a bunch of people invested in the work they do, happy to get you started with it.
+
+Also, it's free. Basically no strings attached (make sure to check the licenses though), you can just straight up use it. So to come back to my point at the beginning of this paragraph. Babylon is wild. But where do you begin?
+
+# Where do you begin
+
+This was honestly a lot harder than I think it should be. I've found that there's a lot of prior knowledge assumed with every tutorial. First of all assuming you have a MacBook without actually stating that. The reason being that if you want to release iOS apps, you have to have some kind of Apple computer. If you're trying this on a windows computer, you may run into issues.
+
+So to get you up to speed enough for you to have some grasp on the concepts. Here's what I think is the most important:
+
+- NPM/Node.js/NPX/NVM
+- node_modules
+- package.json and package-lock.json
+- The `android` and `ios` folders
+
+What's also important but not going to be discussed in this article:
+Eslint or linting in general (https://eslint.org/docs/about/)
+Rollup or other bundling technologies (https://rollupjs.org/guide/en/#overview)
+
+So let's quickly go over these first ones.
+
+# NPM/Node.js/NPX/NVM
+
+I've found the line between these sort of blurred, so here's the difference between them:
+
+- Node.js (https://nodejs.org/en/) is an open-source runtime environment. So this is where your JavaScript runs in.
+- NPM (https://www.npmjs.com/) is the Node Package Manager, so just like you have a package manager in Unity, this is the one for JavaScript and TypeScript. And since it depends on Node.js you first need to get that before you can install libraries in your project. It's possible to globally install libraries, but typically you would have them in the same folder as your project.
+- NPX is a package runner, very similar to NPM. Instead of focussing on the managing of packages, it focusses on running packages instead without you having to worry too much on the versions (a better explanation: https://stackoverflow.com/a/52018825/2497085).
```

Comment:
```suggestion
- NPX is a package runner, very similar to NPM. Instead of focussing on the managing of packages, it focuses on running packages instead without you having to worry too much on the versions (a better explanation: https://stackoverflow.com/a/52018825/2497085).
```